ably-ios/ARTDataEncoder.m

```diff
@@ -0,0 +1,175 @@
+//
+//  ARTDataEncoder.m
+//  ably-ios
+//
+//  Created by Jason Choy on 18/12/2014.
+//  Copyright (c) 2014 Ably. All rights reserved.
+//
+
+#import "ARTCrypto.h"
+#import "ARTLog.h"
+#import "ARTDataEncoder.h"
+
+@interface ARTDataEncoder ()
+
+ART_ASSUME_NONNULL_BEGIN
+
+@property (readonly, nonatomic) ARTLog *logger;
+
+ART_ASSUME_NONNULL_END
+
+@end
+
+@implementation ARTDataEncoder {
+    id<ARTChannelCipher> _cipher;
+    ARTLog *_logger;
```

Comment:
Right, I added the property later and forgot to remove this. Fixed at 7e0be9ca62e2c0c1350aff0f3364b3fbe289cfcd.